doc/faq.rst

```diff
@@ -171,8 +171,8 @@ directory - not the project root. If the library defines a ``name`` distinct fro
 its ``public_name`` then that can be used interchangeably with the ``public_name``
 in this command.
 
-Files and Directories Whose Names Begin with "." (Period) are Ignored by ``source_tree``
-========================================================================================
+Why are files and directories whose names begin with "." (period) are ignored by ``source_tree``?
```

Comment:
We already have "are" at the start.

```suggestion
Why are files and directories whose names begin with "." (period) ignored by ``source_tree``?
```

Tho maybe something like this formulation would be a bit easier to parse out?

```suggestion
Why does ``source_tree`` ignore files and directories when they begin with a "." (period)?
```